homeassistant/helpers/selector.py

```diff
@@ -239,6 +314,10 @@ def validate(e_or_u: str) -> str:
                         f"Entity {e_or_u} belongs to domain {domain}, "
```

Comment:
Are multiple domains support on the frontend or only a single one?

Reply:
Both. https://github.com/home-assistant/frontend/blob/dev/src/data/selector.ts#L107

Reply:
It looks like this validator needs to be updated to support both

Reply:
Nice catch @bdraco 👍